### PR TITLE
Use poetry to manage dependencies

### PR DESCRIPTION
**Pull request by HeitorBoschirolli**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "linguicator-predictor"
+version = "0.0.1"
```
Comment (dmrib) [inline pyproject.toml]: As we discussed, let's start at `0.1.0`.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+description = ""
+authors = ["Heitor Boschirolli <heitor.boschirolli@gmail.com>", "Danilo Ribeiro"]
```
Comment (dmrib) [inline pyproject.toml]: Let's change this to `Machine Grooving Studio <machine.grooving@gmail.com>`? What do you think?

If you agree please change the `mantainers` section as well.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+maintainers = ["Heitor Boschirolli <heitor.boschirolli@gmail.com>", "Danilo Ribeiro"]
+license = "MIT"
+readme = "README.md"
+homepage = ""
```
Comment (dmrib) [inline pyproject.toml]:
```suggestion
homepage = "http://github.com/machinegrooving"
```

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "linguicator-predictor"
+version = "0.0.1"
+description = ""
```
Comment (dmrib) [inline pyproject.toml]:
```suggestion
description = "Text prediction service for Linguiçator"
```

**Comment by HeitorBoschirolli** [comment]
@dmrib applied

**Comment by HeitorBoschirolli** [comment]
should we rename the commit to `Update version to 0.1.0`?

**Pull request merged**